.circleci/config.yml

```diff
@@ -2,7 +2,7 @@
 # syntax, such as anchors, will be fixed automatically.
 version: 2.1
 orbs:
-  shared: getoutreach/shared@dev:local
+  shared: getoutreach/shared@2.14.0
```

Comment:
Should we have a follow up ticket to fix the template https://github.com/getoutreach/stencil-circleci/blob/main/templates/.circleci/config.yml.tpl#L8?

Reply:
Yes.